db/migrate/20220920110309_add_post_ref_to_comment.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddPostRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :post, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Please follow the ERD provided for this tutorial, right now you are not affecting the Author as a reference for the user.